readme.md

```diff
@@ -22,25 +25,94 @@ of the code between the tags for that feature.  Following that, you should remov
 ```
 # TODO: delete me; this is just a reference example
 ```
 
+
+# Feature Descriptions
+
+
+## Django-React
+
+
+### Additional Setup
+
+
+When using this feature, make sure to install the Node.js requirements using the manager of your choice 
+(either `npm install` or `yarn install` will work) before proceeding with development.
+
+
+### Special Consideration for Running
+
+
+For development on localhost when using Django-React, you should run the following command in a separate terminal to
+your standard `runserver` command.
+    
+- `nwb serve --no-vendor`
+
+If you have configured everything correctly, you should see each command complete and notify you
+that the project is ready to be viewed.
+
+
+### Adding a new React component
+
+
+In this paradigm, React components are compiled and injected into the standard Django template. This means we can take 
+advantage of the built-in templating functionality of Django and, with a bit of elbow grease, use the power of React to
+make those templates responsive.
+
+`django-react-loader` uses the same basic pattern for any component:
+
+1. First, ensure that the library is loaded in your template: `{% load django_react_components %}`
+2. Next, ensure that you have rendered the React runtime bundle: `{% render_bundle 'runtime' %}`
+   - Note that you only have to do this once per page where React components will be used. 
+3. Finally, load your React component on the page. `{% react_component 'Home' id='home' %}`
+    - You can add any number of props as named keywords, e.g. `{% react_component 'Home' id='home' prop1=value_from_context %}`
+
+
+### Preparing for deployment
+
+
+To prepare the application for deployment, run `nwb build --no-vendor`. This will generate a `webpack_bundles` folder
+in your `/static` folder populated with the compiled React components. You may now deploy the app as you would any standard
+Django application. 
+
+
+### Other Notes
+
+
+- If you use `nwb serve` in your local development environment, you may see a persistent XHR error in the console -- a 
+request by the app to something like `http://localhost:8000/sockjs-node/info?t=123456789`. This is normal and will 
+  not appear on production or otherwise affect the function of your app. It is an artifact of the context bending we are
+  doing by placing a React component outside the context of its expected Node environment.
+
+- Note that calling `nwb build` does not remove existing compiled data from your static folder -- it may be worth deleting
+`/static/webpack_bundles` before running a build for a deploy, as otherwise your package may become heavier than it
+needs to be.
+   - If you find that the number of files collected by `python manage.py collectstatic` continues to grow, this may be
+    a sign that you should consider deleting the generated files and the `staticfiles` directory and starting with a
+     fresh `python manage.py collectstatic`.
```

Comment:
One of the many reasons I prefer to build as part of the automated deploy script :] I really like that you documented this consideration.